### PR TITLE
Adds some new tips related to lavaland/xenomorphs

### PR DESCRIPTION
**Pull request by improvedname**
**What does this PR do:**
People should really know these things
anyways adds some new tips

**Changelog:**
:cl: Improvedname
add: Adds some new tips
/:cl:

### CODE REVIEW

File: strings/tips.txt [hunk 1]
```diff
@@ -191,6 +194,7 @@ As a Terror Spider, don't forget you can move your nest if the crew is pressurin
 As a Terror Spider, try putting webs on white floor tiles, they are much harder to see there.
 As a Terror Spider, sometimes quickly pulling an enemy into a web can ensure victory in melee combat.
 You can swap floor tiles by holding a crowbar in one hand and a stack of tiles in the other.
+When your restrained to alien nest and theres somebody in a nest next to you, you can unbuckle eachother but not yourself.
```
Comment (Arkatos1) [inline strings/tips.txt]:
```suggestion
When you are restrained to alien nest and there is somebody in a nest next to you, you can unbuckle each other but not yourself.
```

File: strings/tips.txt [hunk 2]
```diff
@@ -122,8 +122,11 @@ As the Librarian, be sure to keep the shelves stocked and the library clean for
 As a Cargo Technician, you can hack MULEbots to make them faster, run over people in their way, and even let you ride them!
 As a Cargo Technician, you can order contraband items from the supply shuttle console by de-constructing it and using a multitool on the circuit board, the re-assembling it.
 As a Cargo Technician, you can earn more cargo points by shipping back crates from maintenance, liquid containers, plasma sheets, rare seeds from hydroponics, and more!
-As a Shaft Miner, the western side of the Asteroid has a lot more rare minerals than on the east, but is a lot more dangerous.
+As a Shaft Miner, northern part of lavaland has a lot more rare minerals than on the southern but is a lot more dangerous.
```
Comment (Markolie) [inline strings/tips.txt]: As a Shaft Miner, the northern part of Lavaland has a lot more rare minerals than the Southern part, but is a lot more dangerous.

File: strings/tips.txt [hunk 3]
```diff
@@ -122,8 +122,11 @@ As the Librarian, be sure to keep the shelves stocked and the library clean for
...
 As a Shaft Miner, always have a GPS on you, so a fellow miner or cyborg can come to save you if you die.
+As a Shaft Miner, you can use certain equipment to be safe from a ash storm, this can range from bluespace shelters to fireproof suits.
```
Comment (Markolie) [inline strings/tips.txt]: As a Shaft Miner, you can use certain equipment to be safe from ash storms. This equipment can range from bluespace shelters to fireproof suits.

File: strings/tips.txt [hunk 4]
```diff
@@ -122,8 +122,11 @@ As the Librarian, be sure to keep the shelves stocked and the library clean for
...
+As a Shaft Miner, consider twice before you take out a tendril, some tendrils may be benifictal for your survival like the legion tendril for legion souls to heal yourself with.
```
Comment (Markolie) [inline strings/tips.txt]: As a Shaft Miner, think twice before taking out a tendril. Some tendrils may be beneficial to your survival. For instance, the Legion tendril can be used to farm Legion souls. These souls can be used to fully heal yourself.

File: strings/tips.txt [hunk 5]
```diff
@@ -122,8 +122,11 @@ As the Librarian, be sure to keep the shelves stocked and the library clean for
...
+As a Ash Walker, you can refine ores with the bonfire
```
Comment (Markolie) [inline strings/tips.txt]: As an Ash Walker, you can refine ores using the bonfire in your nest.

File: strings/tips.txt [hunk 6]
```diff
@@ -191,6 +194,7 @@ As a Terror Spider, don't forget you can move your nest if the crew is pressurin
 As a Terror Spider, try putting webs on white floor tiles, they are much harder to see there.
 As a Terror Spider, sometimes quickly pulling an enemy into a web can ensure victory in melee combat.
 You can swap floor tiles by holding a crowbar in one hand and a stack of tiles in the other.
+When you are restrained to alien nest and there is somebody in a nest next to you, you can unbuckle each other but not yourself.
```
Comment (Markolie) [inline strings/tips.txt]: I don't think this tip should be added - this functionality is more so a bug than intentional behaviour and not something that should really be encouraged.

**Comment by improvedname** [comment]
thank you mr grammar lord markolie

**Comment by Markolie** [comment]
Gotchu fam

File: strings/tips.txt [hunk 7]
```diff
@@ -122,8 +122,11 @@ As the Librarian, be sure to keep the shelves stocked and the library clean for
 As a Cargo Technician, you can hack MULEbots to make them faster, run over people in their way, and even let you ride them!
 As a Cargo Technician, you can order contraband items from the supply shuttle console by de-constructing it and using a multitool on the circuit board, the re-assembling it.
 As a Cargo Technician, you can earn more cargo points by shipping back crates from maintenance, liquid containers, plasma sheets, rare seeds from hydroponics, and more!
-As a Shaft Miner, the western side of the Asteroid has a lot more rare minerals than on the east, but is a lot more dangerous.
+As a Shaft Miner, the northern part of Lavaland has a lot more rare minerals than the Southern part, but is a lot more dangerous.
```
Comment (Citinited) [inline strings/tips.txt]: Southern should be decapitalised

**Comment by improvedname** [comment]
closing/reopening due appveyor

File: strings/tips.txt (hunk 7 above)
Comment (Fox-McCloud) [reply strings/tips.txt]: @improvedname please fix this so I can merge it.

**Comment by Tayyyyyyy** [comment]
One minor thing you could change if you have the time. On line 91, the "flashing blue outline" should be a "flashing green outline".

**Comment by variableundefined** [comment]
@improvedname Please fix the capitalization on line 125 of tips.txt so we can merge the PR

**Comment by Fox-McCloud** [comment]
Been a week. No response. Closing.

**Pull request closed without merging**